scss/jetpack-wizard-banner.scss

```diff
@@ -0,0 +1,110 @@
+@import '../_inc/client/scss/mixin_breakpoint';
+@import '../_inc/client/scss/calypso-colors';
+
+
+.jp-wizard-banner {
+	text-align: left;
+	background: $white;
+	padding: 2rem;
+	box-shadow: 0px 2px 6px rgba( 0, 0, 0, 0.03 ), 0px 1px 2px rgba( 0, 0, 0, 0.03 );
+	border: 1px solid #dcdcde;
+	margin: 40px 20px 0 2px;
+	@include breakpoint( '<660px' ) {
+		margin: 45px 20px 0 2px;
+	}
+}
+
+.jp-wizard-banner-wizard-header {
+	margin-bottom: 1rem !important;
+	margin-top: 1rem !important;
+	font-weight: 500;
+	font-size: 1.5em;
+	line-height: 150%;
+	@include breakpoint( '<660px' ) {
+		text-align: left;
+		font-weight: 400;
+	}
+}
+
+img {
```

Comment:
Maybe we should wrap all our CSS so it's more specific? Targetting just `img` here is a bit dangerous since we'll be enqueuing that CSS on almost all dashboard pages.